.zenodo.json

```diff
@@ -207,6 +212,9 @@
       "name": "Jordan, Kesshi",
       "orcid": "0000-0001-6313-0580"
     },
+    {
+      "name": "Cieslak, Matt"
```

Comment:
@mattcieslak Let me know if you have another name you prefer for publications, your affiliation, and your ORCID. (Sorry if you've given this to me in another package, but I couldn't find it from a quick look.)

Reply:
Here it is @effigies 
```
    {
      "name": "Cieslak, Matthew",
      "affiliation": "Department of Neuropsychiatry, University of Pennsylvania",
      "orcid": "0000-0002-1931-4734"
    },
```

Reply:
Gah, just a few minutes too late! I'll have it in for the 2.4.1 release.